CMakeLists.txt

```diff
@@ -17,6 +17,8 @@ project(dolphin-emu)
 # unique name here.
 set(DISTRIBUTOR "None" CACHE STRING "Name of the distributor.")
 
+set(DOLPHIN_DEFAULT_UPDATE_TRACK "" CACHE STRING "Name of the default update track. If empty, disables auto-update.")
```

Comment:
My assumption is this approach let's you set the value via the command line if you don't want the default?

Reply:
Yes, and it gives you a nice description for the option (just like DISTRIBUTOR).